src/backend/parser/cypher_clause.c

```diff
@@ -283,17 +290,24 @@ static Query *transform_cypher_call_stmt(cypher_parsestate *cpstate,
 static Query *transform_cypher_call_subquery(cypher_parsestate *cpstate,
                                              cypher_clause *clause);
 
+//call...[yield]
+static Query *transform_cypher_call_stmt(cypher_parsestate *cpstate,
+                                      cypher_clause *clause);
+static Query *transform_cypher_call_subquery(cypher_parsestate *cpstate,
+                                          cypher_clause *clause);
```

Comment:
duplicate function header `transform_cypher_call_stmt` and `transform_cypher_call_subquery`